### PR TITLE
Add Open Apply template to the website

### PR DESCRIPTION
**Pull request by dgorodnichy**
# MOTIVATION

Need to add OpenApply case study to the website

# DESCRIPTION
https://trello.com/c/n8uZqzTz/272-add-openapply-use-case-on-the-site

### CODE REVIEW

File: _assets/css/layout/_ai-advantages.scss [hunk 1]
```diff
@@ -25,7 +25,7 @@
   }
 }
 .ai-advantages__description {
-  max-width: 480px;
+  max-width: 580px;
```
Comment (andriyParashchuk) [inline _assets/css/layout/_ai-advantages.scss:28]: Don't change styles for ai page

File: _assets/css/layout/_ai-advantages.scss [hunk 2]
```diff
@@ -41,8 +41,8 @@
   color: #a6a6a6;
 }
 .ai-advantages__img {
-  max-width: 100%;
-  max-height: 100%;
+  max-width: 80%;
```
Comment (andriyParashchuk) [inline _assets/css/layout/_ai-advantages.scss:48]: Don't change styles for ai page

File: _assets/css/layout/_ai-advantages.scss [hunk 3]
```diff
@@ -41,8 +41,8 @@
   color: #a6a6a6;
 }
 .ai-advantages__img {
-  max-width: 100%;
-  max-height: 100%;
+  max-width: 80%;
+  max-height: 80%;
```
Comment (andriyParashchuk) [inline _assets/css/layout/_ai-advantages.scss:49]: Don't change styles for ai page

File: _assets/css/layout/_ai-challenge.scss [hunk 4]
```diff
@@ -50,5 +50,5 @@
   margin: 0 auto;
   width: auto;
   height: auto;
-  max-width: 100%;
+  max-width: 80%;
```
Comment (andriyParashchuk) [inline _assets/css/layout/_ai-challenge.scss:53]: Don't change styles for ai page

File: _assets/css/layout/_ai-projects.scss [hunk 5]
```diff
@@ -16,6 +16,15 @@
     width: 33.33%;
   }
 }
+.ai-projects__story {
+  font-size: 36px;
+  text-decoration: underline;
+  color: #1a8cff;
+  &:hover {
+    color: #1a8cff;
```
Comment (andriyParashchuk) [inline _assets/css/layout/_ai-projects.scss:23]: The same color that in normal state

File: _assets/css/layout/_ai-projects.scss [hunk 6]
```diff
@@ -16,6 +16,15 @@
     width: 33.33%;
   }
 }
+.ai-projects__story {
+  font-size: 36px;
```
Comment (andriyParashchuk) [inline _assets/css/layout/_ai-projects.scss:20]: Add `line-height` also

File: _assets/css/layout/_oa-company-info.scss [hunk 7]
```diff
@@ -0,0 +1,28 @@
+.oa-company-info {
+  line-height: 0px;
+  /* padding: 142px 0 100px; */
```
Comment (andriyParashchuk) [inline _assets/css/layout/_oa-company-info.scss]: Remove commented lines

File: _includes/agentinbox/ai-projects.html [hunk 8]
```diff
@@ -10,16 +10,22 @@ <h2 class="ai-projects__title">
         <span class="fade-down delay-4">Platform</span>
       </a>
     </h2>
+    <br />
```
Comment (andriyParashchuk) [inline _includes/agentinbox/ai-projects.html]: Don't use `<br>`
only `margin-bottom` or `padding-top`

File: _includes/footer.html [hunk 9]
```diff
@@ -29,6 +29,13 @@
         </div>
       </div>
       <div class="cell">
+        <span class='glassdoor-logo-wrapper'>
+        <a class='glassdoor-logo' href='https://www.glassdoor.com/Overview/Working-at-JetThoughts-EI_IE456589.11,22.htm'>
+          <img class="lazy" data-src="{% asset glassdoor-icon.png @path %}" data-srcset="{% asset glassdoor-icon.png @path %}" alt="Glassdoor">
+          <span>Find us</span>
+        </a>
+        </span>
+        <br />
```
Comment (andriyParashchuk) [inline _includes/footer.html]: Don't use `<br>`
only `margin-bottom` or `padding-top`

File: _includes/openapply/oa-projects.html [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+      <span class="fade-down delay-1">Want to see how</span>
+      <span class="fade-down delay-2">Open Apply works?</span>
+    </h2>
+    <h2 class="ai-projects__title">
+      <a href="{{ site.data.use_cases.open_apply.url }}" target="_blank" rel="noreferrer">
+        <span class="fade-down delay-3">Explore Open Apply</span>
+        <span class="fade-down delay-4">Platform</span>
+      </a>
+    </h2>
+    <br />
```
Comment (andriyParashchuk) [inline _includes/openapply/oa-projects.html]: Don't use `<br>`
only `margin-bottom` or `padding-top`

File: _assets/css/layout/_footer.scss [hunk 11]
```diff
@@ -83,3 +83,23 @@
...
+  height: 100%;
+  color: $white;
+  img {
+    height: 30px;
+  }
+}
+.glassdoor-logo:hover {
+  img {
+    opacity: 0.3;
+    filter: alpha(opacity=30);
```
Comment (andriyParashchuk) [inline _assets/css/layout/_footer.scss]: Remove. We don't support IE8

File: _assets/css/layout/_footer.scss [hunk 12]
```diff
@@ -83,3 +83,23 @@
...
+  justify-content: flex-start;
+  align-items: center;
+  flex-direction: row;
+  height: 100%;
+  color: $white;
+  img {
+    height: 30px;
+  }
+}
+.glassdoor-logo:hover {
```
Comment (andriyParashchuk) [inline _assets/css/layout/_footer.scss]: Please use `&:hover`

File: _assets/css/layout/_footer.scss [hunk 13]
```diff
@@ -83,3 +83,23 @@
...
 }
+.glassdoor-logo-wrapper {
+  display: inline-block;
+}
+.glassdoor-logo {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  flex-direction: row;
+  height: 100%;
```
Comment (andriyParashchuk) [inline _assets/css/layout/_footer.scss]: Remove

File: _assets/css/layout/_oa-communication.scss [hunk 14]
```diff
@@ -0,0 +1,69 @@
...
+    position: absolute;
+    content: '';
+    top: 0;
+    right: 0;
+    bottom: 0;
+    background-image: linear-gradient(to bottom,#4b9ce2 0%,#6fcfc1 100%);
+    width: 500px;
+  }
+}
+.oa-communication__description {
```
Comment (andriyParashchuk) [inline _assets/css/layout/_oa-communication.scss:23]: Add a line before selector

File: _assets/css/media/_laptop.scss [hunk 15]
```diff
@@ -286,17 +290,23 @@
...
-    .ai-communication__img {
+  .ai-go-it,
+  .oa-go-it {
+    .ai-communication__img,
+    .oa-communication__img {
       right: -16.84%;
       max-width: 86.31%;
     }
     &.animation {
+      .oa-communication__img {
```
Comment (andriyParashchuk) [inline _assets/css/media/_laptop.scss]: .ai-communication__img,      
.oa-communication__img {

File: _assets/css/media/_laptop.scss [hunk 16]
```diff
@@ -286,17 +290,23 @@
...
+        top: -11px;
+      }
       .ai-communication__img {
         top: -11px;
       }
     }
   }
+
```
Comment (andriyParashchuk) [inline _assets/css/media/_laptop.scss]: Remove empty line or add empty line before all selectors
Let's use a single standard

**Comment by andriyParashchuk** [comment]
Large font size and small line-height - image

Broken style for ai page - 
image
image

**Comment by andriyParashchuk** [comment]
@dgorodnichy Please check my comments

File: _assets/css/layout/_footer.scss [hunk 17]
```diff
@@ -83,3 +83,24 @@
...
+  align-items: center;
+  flex-direction: row;
+  color: $white;
+  img {
+    height: 30px;
+  }
+  &:hover {
+    img {
+      opacity: 0.3;
+      filter: alpha(opacity=30);
```
Comment (andriyParashchuk) [inline _assets/css/layout/_footer.scss]: Has not been removed

**Review by andriyParashchuk — APPROVED** [review]
LGTM

**Comment by andriyParashchuk** [comment]
@dgorodnichy One moment. Please add ItemList schema OA page here https://github.com/jetthoughts/jetthoughts.github.io/blob/develop/_includes/head.html#L100
Like AI page

**Comment by pftg** [comment]
Deployed to https://jetthoughts-pr-421.surge.sh

**Pull request merged**